### PR TITLE
new Dagon lobby screen 

### PR DESCRIPTION
**Pull request by CHEtheGREAT**
https://i.imgur.com/gIU5nBZ.png

Adds new lobby screen I made a while ago ( was approved )

I'm new to git, this is my first PR, so i apologize if i didn't do it correctly i'm kinda just learning by trying it out

### CODE REVIEW

File: maps/torch/torch_lobby.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /datum/map/torch
-	lobby_screens = list('maps/torch/lobby/lobby_screen_test2.gif')
+	lobby_screens = list('maps/torch/lobby/hestialobbysmall.png')
```
Comment (MarinaGryphon) [inline maps/torch/torch_lobby.dm:2]: I don't think you want the small one, it might be too small and leave a lot of letterboxing.

Comment (CHEtheGREAT) [reply maps/torch/torch_lobby.dm:2]: I was told by Mazian to downscale it to 400x400px, is there a size i should shoot for ? i have the original AI file so i can size it to whatever size is needed

Comment (MarinaGryphon) [reply maps/torch/torch_lobby.dm:2]: In that case, it'll probably work.

**Comment by MarinaGryphon** [comment]
Scaling seems a bit odd. https://i.imgur.com/L1VqBxT.png A bit fuzzy, I think it might be too small?

**Comment by CHEtheGREAT** [comment]
Right, I can upload any size but I have no clue how in game scaling would look. What size should I upload it as?

**Comment by CHEtheGREAT** [comment]
it works fine now :)

**Comment by CHEtheGREAT** [comment]
Please ignore all of this, new to git and i had some issues with branches and stuff

ill redo this PR momentarily

**Pull request closed without merging**